pyanaconda/core/configuration/anaconda.py

```diff
@@ -363,7 +363,8 @@ def set_from_opts(self, opts):
         # Set the storage flags.
         self.storage._set_option("dmraid", opts.dmraid)
         self.storage._set_option("ibft", opts.ibft)
-        self.storage._set_option("gpt", opts.gpt)
+        if hasattr(opts, "gpt"):
```

Comment:
I did some testing and it looks like you could use `if "gpt" in opts`. It seems easier to understand.